lib/Yield/ControlFlow/SVG.cpp

```diff
@@ -264,29 +257,31 @@ defaultArrowHeads(const yield::cfg::Configuration &Configuration) {
     return duplicateArrowHeadsImpl(8, 3, 2);
 }
 
-static std::string exportCFG(const yield::Graph &Graph,
-                             const yield::Function &Function,
-                             const model::Binary &Binary,
-                             const yield::cfg::Configuration &Configuration) {
+template<bool ShouldEmitEmptyNodes, typename CallableType>
+static std::string exportGraph(const yield::Graph &Graph,
+                               const yield::cfg::Configuration &Configuration,
+                               CallableType &&Callable) {
```

Comment:
* Find a more suitable name for this argument and make a concept.
* Are you 100% you won't need more than one method? Consider making a virtual class.

Reply:
1. Yeah, sorry. This method just converts `MetaAddress` of a node in the graph into its contents (ptml or otherwise). This lets me reuse all the graph-creation logic between different graph types.

2. This is a highly-specialized helper, local to this compilation unit. Since it never touches any actual interfaces, it can be freely changed at any point. But to answer the question, I can't imagine it needing more customizability than that, unless we try to do something really fancy with our graphs.

Reply:
Something like this? https://github.com/revng/revng/pull/191/commits/68fb72a9e97859c1d3eae031b2d074d7cc694c4a